### DISCUSSION TITLE
ADempiere Steering meeting August 12th, 2022

### DISCUSSION THREAD

**Discussion by rvergara** [adempiere/adempiere#3955] [Steering Meetings]
### Meeting Info
**Date:** August 12th, 2022
**Time:** 19:00 GMT ~ 20:00 GMT
**Location**: Discord meeting room
**Meeting URL:** None
**Recorder:** No
**Meeting Objetive:** Review of Project and admin activities to be performed
### **Participants**
@rvergara 
@niclopup 
@e-Evolution 
@eduardolopezuy 
@yamelsenih 
@marcalwestf 
### **Agenda**

1. Progress ADempiere web site
2. Knowledge base tools discussions
3. 3.9.4 release calendar discussion
4. Discord channel management
5. Others
### **Points Discussied**

- ADempiere web site in spanish is in progress and it is expected to be completed this week (Friday 19th)
- It was agreed that as sonn as the web page is spanish is complete, 3.9.4 version will be realesed without adding any further additional developments
- a discussion of Nicolas Lopez proposal for discord channel management was performed. During the meeting a set of channels was agreed and documented by Nicolas. Decision was agreed for Nicolas to proceed to configure the Spanish and English Discord channels as per the discussed proposal
- It was agreed that @rvergara would be the initial moderator for the discord channel, @niclopup will configure these privileges accordingly

### **Next Meeting**
August 26th. 19:00 GMT, Discord meeting room (proposed)

**Comment by e-Evolution** [comment]
Hi community 

1.- About the new page and website, I think it should be ready together with the release of ADempiere, I think the release should be synchronized with the page in English and Spanish when are is ready.

2.- I am of the idea that the most important thing in our knowledge base is the information and data, that is to say, that regardless of the solution chosen we cannot depend on a company or solution, this is very important for us as a community, we have the experience of Sourceforge a lot of knowledge of Adempiere stayed there and we did not have an easy or quick way to preserved it, the most important thing about this point is that we need the control over the information in the knowledge database so that if you ask me, we would be developed in Adempiere as a module based on the initial work of Compiere, I know it's more work but the reward is higher.

3.- With the understanding that we must synchronize the launch with the website, we will have to see how both timelines are.

They would like to release in this version ADempiere Webui ZK 9.6 almost finished the migration requires help for tests and well whoever joins can also support in fixing what comes out, it's a matter of putting your shoulder to it, I leave it only as a proposal if so to share the branch and I need committed volunteers to do the tests, I'm on this full time now with this work.

4.- The most important thing about the community in discord is that it has an international focus so that languages ​​are not mixed, the structure can be any but it should be the same for English and Spanish, I would start in a minimalist way and focus it In two big topics, the technical and the functional as categories and then in the functional, I agree with Mario that it be for functionality as in GitHub and in the technical the minimum necessary per technical topic (DB, UI, Code, Architecture).


kind regards
Victor
